tensorflow/contrib/periodic_intersperse/python/kernel_tests/periodic_intersperse_op_test.py

```diff
@@ -0,0 +1,43 @@
+# =============================================================================
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+import numpy as np
+import tensorflow as tf
+from tensorflow.contrib.periodic_intersperse import periodic_intersperse as pi
+
+
+class PeriodicIntersperseTest(tf.test.TestCase):
+    def testPeriodicIntersperse(self):
+
+        # basic 2-D tensor
```

Comment:
the two tested that are already there effectively test the 1D case. I will write much more detailed tests for 2D and 3D cases tomorrow.